middleware/validateToken.js

```diff
@@ -0,0 +1,8 @@
+const jwt = require("jsonwebtoken");
```

Comment:
To the best of my understanding `validateToken` is used as a helper function. Therefore, I would suggest you remove it from the middleware function.